decidim-admin/lib/decidim/admin/form_builder.rb

```diff
@@ -60,6 +60,11 @@ def autocomplete_select(attribute, selected = nil, options = {}, prompt_options
         template += error_for(attribute, options) if error?(attribute)
         template.html_safe
       end
+
+      # Calls Decidim::FormBuilder#editor with default options for admin.
+      def editor(name, options = {})
+        super(name, options.merge(toolbar: :full, lines: 25))
```

Comment:
Hi @aitorlb @decidim/core! This line overwrites whatever setting we pass, so for example we can't change the number of lines, because this line will always set it to 25. I'd like to change it in #5631 so that it looks like this:

```ruby
super(name, { toolbar: :full, lines: 25}.merge(options))
```

This would still set a default value, and let us overwrite those defaults whenever we need to. Does it sound good to you? 😄

Reply:
Hi @mrcasals! I'm no longer involved in `decidim`. Next time, I would recommend to just refactor the code in your PR since you know what you're doing  😄. Good catch btw!

Reply:
Go ahead @mrcasals !

Reply:
Ok, thanks!